ci/check-protobuf-against-stable.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+readonly SRC_DIR="$(dirname "${SCRIPT_DIR}")"
```

Comment:
I'd find
```suggestion
readonly PROJECT_ROOT="$SCRIPT_DIR/.."
```
a lot easier to read here.

Reply:
Done, e3cb71a